client/src/Components/Customer/Service/Services.tsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react'
+import "./Service.scss"
+import API from "../../../API"
+import { useEffect, useState } from "react"
+import { Service } from '../../../Models/service'
+import { Button, Card } from 'react-bootstrap';
+
+function GetServices() {
+
+    const [serviceList, setServiceList] = useState<Service[]>([]);
+    const [error, setError] = useState<string>("");
+    const [ticket, setTicket] = useState<string>("");
+    const [ticketIsRequested, setticketIsRequested] = useState<boolean>(false);
+    const [serviceId, setServiceId] = useState<number>(0);
```

Comment:
are we sure that there is a 0 service? Whenever the page refresh it sends 2 calls to get service 0